### PR TITLE
Implement comments Continuation

### PR DESCRIPTION
**Pull request by powermukisa**
Continue with comments implementation for Features:
- Add ability to create, edit and delete a Feature comment
- Fix the collapse and show content buttons in SIdeBar (they need styling)
- Add report reducers and actions for report comments
- Update Gezako login mantra/motto
- Update the SideBar Text

### CODE REVIEW

File: src/components/comments/CreateComment.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+import * as firebase from "firebase";
+// import * as firebase from "firebase";
+
+const CreateComment = (props) => {
+  const { user } = props;
+  const [commentTextAreaActive, setCommentTextAreaActive] = useState(false);
+
+  const [text, setText] = useState('');
+
+
```
Comment (Codebreake) [inline src/components/comments/CreateComment.js]: We could get rid of this extra line

File: src/store/actions/reportActions.js [hunk 2]
```diff
@@ -379,3 +382,98 @@ export const updateServiceStats = (service, serviceStats) => {
...
+  }
+};
+
+export const resetGetSpockReportComments = () => {
+  return (dispatch) => {
+    dispatch({type: 'RESET_GET_SPOCK_REPORT_COMMENTS'});
+  }
+};
+
+
```
Comment (Codebreake) [inline src/store/actions/reportActions.js]: extra line not neccesary

File: src/components/comments/Comment.js [hunk 3]
```diff
@@ -1,22 +1,16 @@
...
+      createdAt,
+      text
+      ) {
+    this.authorId = authorId;
+    this.authorName = authorName;
+    this.authorPhotoUrl = authorPhotoUrl;
+    this.createdAt = createdAt;
+    this.text = text;
+  }
+}
```
Comment (Codebreake) [inline src/components/comments/Comment.js]: new line

File: src/components/comments/CreateOrEditComment.js [hunk 4]
```diff
@@ -1,35 +1,83 @@
...
+// import * as firebase from "firebase";
 
 const CreateOrEditComment = (props) => {
   const { user } = props;
-  const [isActive, setState] = useState(false);
+  const [commentTextAreaActive, setCommentTextAreaActive] = useState(false);
+
+  const [text, setText] = useState('');
+
+
```
Comment (Codebreake) [inline src/components/comments/CreateOrEditComment.js]: extra line not needed

File: src/components/comments/Comment.js (hunk 3 above)
Comment (Codebreake) [reply src/components/comments/Comment.js]: what does the outdated label besides comment.js mean?

File: src/components/comments/EditComment.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+      props.comment.text = text;
+      props.comment.updatedAt = firebase.firestore.Timestamp.fromDate(new Date());
+
+      updateFeatureComment(props.featureId, comment);
+
+      props.setShowEditComment(false) //hide the edit component
+    }
+  };
+
+
```
Comment (Codebreake) [inline src/components/comments/EditComment.js]: extra line not needed

File: src/store/actions/reportActions.js [hunk 6]
```diff
@@ -379,3 +382,99 @@ export const updateServiceStats = (service, serviceStats) => {
...
+    .doc(commentId)
+    .delete().then(
+        result => {
+          dispatch({type: 'DELETE_SPOCK_REPORT_COMMENT_SUCCESS'});
+        }, err => {
+          dispatch({type: 'DELETE_SPOCK_REPORT_COMMENT_ERROR', error: err});
+        }
+    );
+  }
+};
```
Comment (Codebreake) [inline src/store/actions/reportActions.js]: new line needed

Comment (Codebreake) [reply src/store/actions/reportActions.js]: oh this was fixed  in the last commit 'add new lines at EOF ' .ignore

**Comment by Codebreake** [comment]
All other files LGTM

File: src/components/comments/Comment.js (hunk 3 above)
Comment (powermukisa) [reply src/components/comments/Comment.js]: it means that I fixed the issue and pushed before you posted your comment. so your comment was stale

Comment (Codebreake) [reply src/components/comments/Comment.js]: go it

**Pull request merged**